.github/workflows/docker/buildwheel.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+DOCKER_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source "$DOCKER_DIR/shared.env"
+
+set -e -x
+
+ARCH=`uname -p`
+echo "arch=$ARCH"
+curl -o gsl-${GSL_VERSION}.tar.gz "ftp://ftp.gnu.org/gnu/gsl/gsl-${GSL_VERSION}.tar.gz"
+tar -zxf gsl-${GSL_VERSION}.tar.gz
+cd gsl-${GSL_VERSION}
+./configure --prefix=/usr
+make -j 2
+make install
+cd ..
+
+for V in "${PYTHON_VERSIONS[@]}"; do
+    PYBIN=/opt/python/$V/bin
+    rm -rf build/       # Avoid lib build by narrow Python is used by wide python
+    # Instead of letting setup.py install a newer numpy we install it here
+    # using the oldest supported version for ABI compatibility
+    $PYBIN/pip install oldest-supported-numpy
+    cat /usr/include/gsl/gsl_version.h
+    STATIC_GSL=1 $PYBIN/python setup.py build_ext --inplace
```

Comment:
Actually, do we need to do the static build thing? Doesn't ``auditwheel repair`` do some magic where it finds the external libs and copies them into the wheel?